tests/regression/runner.py

```diff
@@ -192,17 +192,22 @@ def test_mam_recv_msg(self):
         #    2. Unicode msg [failed] {\"message\":\"Internal service error\"}
         #    3. Not existing bundle hash (address)
         query_string = [
-            "BDIQXTDSGAWKCEPEHLRBSLDEFLXMX9ZOTUZW9JAIGZBFKPICXPEO9LLVTNIFGFDWWHEQNZXJZ9F9HTXD9",
+            "WRLPYSQSDIFAEDKTPKNTMMYLCPJXKBNRL9SQOQPYIXLSGVKH9RRUALUPZNEHWVCMZJFQURTXSONKTWOIE",
             "", "生れてすみません"
         ]
 
-        expect_cases = ["\"message\":\"ToBeOrNotToBe\""]
+        expect_cases = ["\"message\":\"multiTest\""]
```

Comment:
Is is necessary to change the message?

Reply:
I found this relates to some other big issue, so I open a new issue #264 to fix this.